content/docs/ops/design/secrets.md

```diff
@@ -158,7 +158,40 @@ Concourse, and CredHub.
 
 ## Bootstrapping CredHub
 
-> To Be Decided
+Deploying anything  before there is a CredHub available on the BOSH director
+you're deploying to requires you to deploy CredHub manually using established
+`operations` and `variables` files that leverage BOSH interpolate.
+
+Ensure that you are creating a database with `rds.force_ssl` in its parameter
```

Comment:
Our bootstrap process should encode this options with code. Not sure we also need to document it.

Reply:
yeah good point. I was included it in here as a design document, but I've since removed it.